### PR TITLE
Add sidebar permissions to allow turning groups on/off globally

### LINKED ISSUE #276: Sidebar "Content" group missing if user doesn't have Page permissions
### Summary of issue
The sidebar group "Content" disappears if the user doesn't have permissions for 'page.*', even if they have permissions for another module that has an entry in that group.

### Steps to reproduce
Create/edit a user, set 'page.*' permissions to Deny.

Looks like it is caused by this block of code:
```php
$group->authorize(
    $this->auth->hasAccess('page.*')
);
```
here: [Modules/Page/Sidebar/SidebarExtender.php#L50-L52](https://github.com/AsgardCms/Platform/blob/2.0/Modules/Page/Sidebar/SidebarExtender.php#L50-L52)

Edit: To clarify, this only happens if the user doesn't have ANY permissions for the Page module.

### System Details

- Operating System: Mac/Linux
- PHP Version: 7.0.11
- Laravel Version: 5.3.18
- `Asgardcms/Platform`: 2.0.x-dev

### PR DESCRIPTION
**Pull request by mikemand**
Closes #276

- Add missing language line for workshop themes

- Add missing permissions from AsgardCms/Workshop#31 to permissions seed

- Add Workshop permission to globally turn on/off sidebar group

- Remove permissions check on core sidebar group

- Sort permission seeder in order of permissions config file

### CODE REVIEW

File: Modules/Core/Sidebar/SidebarExtender.php [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?php
 
-namespace Modules\Page\Sidebar;
+namespace Modules\Core\Sidebar;
```
Comment (mikemand) [inline Modules/Core/Sidebar/SidebarExtender.php]: You didn't see that. 👀

Comment (armababy) [inline Modules/Core/Sidebar/SidebarExtender.php]: :see_no_evil:

**Comment by nWidart** [comment]
I tried this out and:

- permissions file was not published
- permission did not show on roles/users permissions tab (probably because file was not published)

**Comment by nWidart** [comment]
Indeed adding:

``` php
$this->publishConfig('core', 'permissions');
```

To the core service provider did the trick to publish the file

**Pull request merged**